x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/action/SearchApplicationSearchRequest.java

```diff
@@ -122,4 +129,19 @@ public String[] indices() {
     public IndicesOptions indicesOptions() {
         return IndicesOptions.strictNoExpandForbidClosed();
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startObject();
+        builder.field(NAME_FIELD.getPreferredName(), name);
+        if (queryParams.isEmpty() != true) {
```

Comment:
We might be able to use `builder.map` or `builder.mapContents` here?

If we can't: 
```suggestion
        if (queryParams.isEmpty() == false) {
```

Reply:
Went with Suggested change:  af753f213ae0513cd019ad628cadbedc1ab7ce08